baseapp_auth/tests/graphql/test_queries_user.py

```
@@ -169,3 +184,30 @@ def test_anon_can_query_users_list_with_optimized_query_with_public_id(graphql_c
+    assert len(content["data"]["users"]["edges"]) == 1
+
+
+def test_anon_can_query_users_list_with_filter_by_profile_name(graphql_client_with_queries):
+    UserFactory.create_batch(2)
+    user_1 = UserFactory()
+    user_1.profile.name = "test"
+    user_1.profile.save()
+    user_2 = UserFactory()
+    user_2.profile.name = "testing"
+    user_2.profile.save()
+    response, queries = graphql_client_with_queries(
```

Ruff (0.14.14)

[warning] Unpacked variable `queries` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)